src/main/java/school/lemon/changerequest/java/introduction/hw2/Task1.java

```diff
@@ -1,7 +1,9 @@
 package school.lemon.changerequest.java.introduction.hw2;
 
+import java.util.Formatter;
+
 public class Task1 {
     public static String makeTags(String tag, String text) {
-        return "";
+        return "<" + tag + ">" + text + "<" + "/" + tag + ">";
```

Comment:
I'd also mention, that it's the worst possible implementation (IMO).
I'd better see here usage of `StringBuilder` or `String.format()`